src/pgrn-compatible.h

```diff
@@ -95,6 +95,12 @@ typedef Oid PGrnRelFileNumber;
 #	define pg_noreturn
 #endif
 
+#if PG_VERSION_NUM < 180000
+#	define COMPARE_LT BTLessStrategyNumber
+#	define COMPARE_GT BTGreaterStrategyNumber
+#	define pk_cmptype pk_strategy
```

Comment:
The name has changed from PostgreSQL18.
https://github.com/postgres/postgres/commit/8123e91f